packages/builder/src/stores/builder/components.ts

```diff
@@ -213,8 +212,10 @@ export class ComponentStore extends BudiStore<ComponentState> {
     }
 
     const def = this.getDefinition(enrichedComponent?._component)
-    const filterableTypes = def?.settings?.filter(setting =>
-      setting?.type?.startsWith("filter")
+    const filterableTypes = def?.settings?.filter(
+      setting =>
+        setting?.type?.startsWith("filter") &&
+        setting?.type !== "filterConfiguration"
```

Comment:
I think it would make sense to move to a whitelist here to keep things more maintainable. So we just need to include `filter` and `filter/relationship`.